chatter.js

```diff
@@ -1,12 +1,80 @@
 Chatter = {
-    options: {
-        messageLimit: 30,
-        nickProperty: "username",
+  options: {
+    messageLimit: 30,
+    nickProperty: "username",
+  }
+};
+
+Chatter.configure = function (opts) {
+  _.extend(this.options, opts);
+};
+
+/**
+ * @summary Adds user to Chatter and defines its privileges
+ * @locus Server
+ * @param {string} userId Unique string identifying the users of your application.
+ * @param {string} userType Defines the restricitons on the new user. Can either be set to "admin" or "standard".
+ * @returns {string} userId
+ */
+Chatter.addUser = function(userId, userType) {
+   check(userId, String);
+   check(userType, Match.Maybe(String));
+  const user = Meteor.users.findOne({_id: userId});
+
+  Chatter.User.upsert({
+    userId: userId
+  }, {
+    $set: {
+      userType: userType,
+      nickname: getNickname(user)
     }
+  });
+  return Chatter.User.findOne({userId: userId})._id;
 };
 
+/**
+ * @summary Adds empty room to Chatter.
+ * @locus Server
+ * @param {string} rommName
+ * @returns {string} roomId
+ */
+Chatter.addRoom = function(roomName) {
+  check(roomName, String);
 
-Chatter.configure = function (opts) {
-    _.extend(this.options, opts);
+  return Chatter.Room.insert({name: roomName});
 };
 
+/**
+ * @summary Adds user to room.
+ * @locus Server
+ * @param {string} userId Unique string identifying user.
+ * @param {string} roomName Unique string identifying the room.
+ * @returns {string} userRoomId
+ */
+Chatter.addUserToRoom = function(userId, roomId) {
+  check(userId, String);
+  check(roomId, String);
+  const chatterUserId = Chatter.User.findOne({userId: userId})._id;
```

Comment:
Added a checker that throws appropriate error before trying to access attributes. 
Regarding errors caused by mistakes in parameters, all responsibility for handling is always given to client right? Meaning they are the ones that need to surround method calls with try-catch block or callbacks.